libs/etterlatte-behandling-model/src/main/kotlin/no/nav/etterlatte/libs/common/behandling/BehandlingInfo.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.etterlatte.libs.common.behandling
+
+import no.nav.etterlatte.libs.common.grunnlag.Grunnlagsopplysning
+import java.time.LocalDate
+import java.util.UUID
+
+enum class Aldersgruppe {
```

Comment:
Det her er allerede gjort på main, virker mye lettere starte derifra også bare legge det på detaljertbehandling så er du ferdig. For såvidt jeg ser er frontend endringene bare renaming? Er dette forarbeidet til pensjonsbrev eller hva hvorfor legges de på detaljertbehandling?